### PR TITLE
Issue #289: Increase the specificity of the .error background-color t…

### PR DESCRIPTION
**Pull request by serundeputy**
This fixes #289.

### CODE REVIEW

File: core/modules/system/css/system.theme.css [hunk 1]
```diff
@@ -141,17 +141,14 @@ abbr.ajax-changed {
...
   border: 2px solid red;
 }
-.form-type-checkboxes.error,
-.form-type-radios.error {
+.form-type-checkboxes.form-error,
+.form-type-radios.form-error {
   border: 2px solid red;
 }
 /* Display border on a single checkbox, but not on checkboxes within a group. */
 .form-type-checkbox.error {
```
Comment (quicksketch) [inline core/modules/system/css/system.theme.css]: Looks like this line still need to be updated as well.

Comment (serundeputy) [reply core/modules/system/css/system.theme.css]: I removed this line as it is uneeded now.

**Comment by quicksketch** [comment]
Closing for new sandbox.

**Comment by serundeputy** [comment]
@quicksketch this looks good to me now.  permission to merge please.

**Comment by quicksketch** [comment]
Go for it @serundeputy :)

**Comment by quicksketch** [comment]
Ah well, I couldn't wait so I've merged this into 1.x and 1.5.x. Thanks @serundeputy!

**Pull request merged**